themes/theme-ios11/theme-config.js

```diff
@@ -10,12 +10,22 @@ const {
  * primaryContrast or accentContrast. To avoid breaking of this system, the required colors
  * are taken from the app-config and injected here.
  */
+/** @type {ThemeConfig} */
 module.exports = {
   colors: {
     primary,
     accent,
     light,
     dark,
+    /** Backward compatibility for newly added extension config values */
+    lightTransparent: 'rgba(255, 255, 255, 0.82)',
+    lightDark: '#323232',
+    overlay: '#fafafa',
+    lightOverlay: '#fafafa',
+    darkOverlay: '#f0f0f0',
+    gray: '#747474',
+    lightGray: '#9a9a9a',
```

Comment:
Please respect the comments from `theme-gmd/extension-config.json` here as well!